src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -242,6 +243,46 @@ private bool InternalTryReceiveOne(out MessageEnvelope envelope, TimeSpan? max,
             return false;
         }
 
+        private async Task<MessageEnvelope> InternalTryReceiveOneAsync(TimeSpan? max, CancellationToken cancellationToken, bool shouldLog)
```

Comment:
`out` keyword not supported in `async/await`. Instead of returning a `boolean` the message itself is returned and if the message is null, we know we did not take! (`didTake`)